distributions/demo-server/build.gradle

```diff
@@ -197,24 +195,18 @@ task copyWebApp {
     }
 }
 
-task buildClientImage(type: DockerBuildImage) {
-    description = 'Builds the Client Docker image'
-    dependsOn shadowJar, createClientDockerfile, copyWebApp
-    dockerFile = project.file("${buildDir}/client/Dockerfile")
+task dockerize(type: DockerBuildImage) {
+    description = 'Builds the Demo Docker image including API server and nginx'
+    dependsOn shadowJar, createDockerfile, copyWebApp
+    dockerFile = project.file("${buildDir}/demo/Dockerfile")
     inputDir = project.file(".")
-    tag = 'dataportability/client:latest'
-}
-
-task dockerize {
-    description = 'Builds the Client and API Server Docker images'
-    group = 'docker'
-    dependsOn buildDemoServerImage, buildClientImage
+    tag = 'datatransferproject/demo:latest'
 }
 
 task installToolsAndDockerize {
-    description = 'Installs required build tools and builds the Client and API Server Docker images'
+    description = 'Installs required build tools and builds the Client and API Server Docker image'
```

Comment:
sometimes we say client, sometimes we say nginx, to refer to the same thing. I would use the same term consistently to be clear, or come up with a new one like "demo-frontend" - nginx serving Angular

Reply:
Done.